siac/misccmd.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/NebulousLabs/Sia/sia"
+)
+
+var (
+	updateCmd = &cobra.Command{
+		Use:   "update [check|apply]",
+		Short: "Update Sia",
+		Long:  "Check for (and/or download) available updates for Sia.",
+		Run:   wrap(updatecmd),
+	}
+
+	updateCheckCmd = &cobra.Command{
+		Use:   "update check",
+		Short: "Check for available updates",
+		Long:  "Check for available updates.",
+		Run:   wrap(updatecheckcmd),
+	}
+
+	updateApplyCmd = &cobra.Command{
+		Use:   "update apply [version]",
+		Short: "Apply an update",
+		Long:  "Update Sia to 'version'. To use the latest version, run 'update apply current', or simply 'update'.",
+		Run:   wrap(updateapplycmd),
+	}
+
+	statusCmd = &cobra.Command{
+		Use:   "status",
+		Short: "Print the current state of the daemon",
+		Long:  "Query the daemon for values such as the current difficulty, target, height, peers, transactions, etc.",
+		Run:   wrap(statuscmd),
+	}
+
+	stopCmd = &cobra.Command{
+		Use:   "stop",
+		Short: "Stop the Sia daemon",
+		Long:  "Stop the Sia daemon.",
+		Run:   wrap(stopcmd),
+	}
+
+	syncCmd = &cobra.Command{
+		Use:   "sync",
+		Short: "Synchronize with the network",
+		Long:  "Attempt to synchronize with a randomly selected peer.",
+		Run:   wrap(synccmd),
+	}
+)
+
+// TODO: this should be defined outside of siac
```

Comment:
siad doesn't define it, instead just using a struct literal. Later on I plan to consolidate+export some of the API stuff to make API code cleaner in both siad and siac.